### ISSUE TITLE
Refactor to TypeScript

### ISSUE THREAD

**Issue by neopostmodern** [neopostmodern/wohnungsbot#102]
Flow has become an unbearable liability on multiple levels

**Comment by neopostmodern** [comment]
Tool ideas:
- https://www.npmjs.com/package/babel-plugin-flow-to-typescript
- https://github.com/Khan/flow-to-ts

**Comment by geohiehoe** [comment]
```
    npm install typescript --save-dev
    npm install @babel/preset-typescript --save-dev
    npm uninstall @babel/preset-flow

    npm install -g @khanacademy/flow-to-ts
    flow-to-ts **/*.js --write --delete-sources

**Comment by neopostmodern** [comment]
Closed with PR #185
